syntax = "proto3";

package mysimplerpc;

// protoc rpcheader.proto --cpp_out=./

message RpcHeader {
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
}